### PR TITLE
Mobileapps 1143

### PR DESCRIPTION
**Pull request by ankitgoyalgloballogic**

### CODE REVIEW

**Review by aman-alfresco — DISMISSED** [review]

File: ContentApp/BusinessLayer/Action Menu/ViewModels/NodeActionsViewModel.swift [hunk 1]
```diff
@@ -30,13 +30,18 @@ protocol NodeActionsViewModelDelegate: AnyObject {
...
+}
+
 class NodeActionsViewModel {
     private var node: ListNode?
     private var actionFinishedHandler: ActionFinishedCompletionHandler?
     private var coordinatorServices: CoordinatorServices?
     private let nodeOperations: NodeOperations
     private let listNodeDataAccessor = ListNodeDataAccessor()
     weak var delegate: NodeActionsViewModelDelegate?
+    var moveDelegate: NodeActionMoveDelegate?
```
Comment (gl-lovekesh) [inline ContentApp/BusinessLayer/Action Menu/ViewModels/NodeActionsViewModel.swift:44]: make it weak.

File: ContentApp/BusinessLayer/Lists/Models/OfflineFolderDrillModel.swift [hunk 2]
```diff
@@ -136,7 +136,8 @@ extension OfflineFolderDrillModel: EventObservable {
             }
         case .restore:
             delegate?.needsDataSourceReload()
-
+        case .moveToFolder:
```
Comment (gl-lovekesh) [inline ContentApp/BusinessLayer/Lists/Models/OfflineFolderDrillModel.swift:139]: Merge the same cases.

File: ContentApp/PresentationLayer/Components/SystemSearch+ListActions.swift [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+
+extension SystemSearchViewController: NodeActionsViewModelDelegate,
```
Comment (gl-lovekesh) [inline ContentApp/PresentationLayer/Components/SystemSearch+ListActions.swift:21]: Clean this file, Remove unwanted code.

**Review by gl-lovekesh — CHANGES_REQUESTED** [review]

File: ContentApp/BusinessLayer/Action Menu/ViewModels/NodeActionsViewModel.swift (hunk 1 above)
Comment (ankitgoyalgloballogic) [reply ContentApp/BusinessLayer/Action Menu/ViewModels/NodeActionsViewModel.swift:44]: Done

File: ContentApp/BusinessLayer/Lists/Models/OfflineFolderDrillModel.swift (hunk 2 above)
Comment (ankitgoyalgloballogic) [reply ContentApp/BusinessLayer/Lists/Models/OfflineFolderDrillModel.swift:139]: Done

File: ContentApp/PresentationLayer/Components/SystemSearch+ListActions.swift (hunk 3 above)
Comment (ankitgoyalgloballogic) [reply ContentApp/PresentationLayer/Components/SystemSearch+ListActions.swift:21]: Done

**Review by gl-lovekesh — APPROVED** [review]

**Pull request merged**